src/market/_internal/queue.py

```diff
@@ -37,6 +39,30 @@ def __parse_queue_item(buffer: Sequence[int], queue_type: QueueType) -> Union[Ev
     layout = EVENT_LAYOUT if queue_type == QueueType.Event else REQUEST_LAYOUT
     parsed_item = layout.parse(buffer)
     parsed_item.pop("_io")  # Hack: Drop BytesIO object to fit kwargs into Event/Request object.
+    if queue_type == QueueType.Event:  # pylint: disable=no-else-return
+        return Event(
```

Comment:
Okay. Since we're doing this. We might as well consolidate some duplicate fields between event and request and also remove the container object from `event_flags` and `request_flags`.

Reply:
added `event_flags` and `request_flags`.

Reply:
Sorry. This is not what I meant. I mean to replace to Container type here:
https://github.com/serum-community/pyserum/blob/444f89d8995583b252d5ee54319c453d6631817b/src/market/types.py#L110

Reply:
I believe this has been changed.